test/errors-test.js

```diff
@@ -73,4 +74,20 @@ describe('Errors', () => {
     await assert.isRejected(iamap.create(devnull, { hashAlg: 0x00 /* 'identity' */, bitWidth: 10, bucketSize: 16 }, new Uint8Array(2)))
     await assert.isFulfilled(iamap.create(devnull, { hashAlg: 0x00 /* 'identity' */, bitWidth: 10, bucketSize: 16 }, new Uint8Array((2 ** 10) / 8)))
   })
+
+  it('test abort signal', async () => {
+    const store = memoryStore()
+    const controller = new AbortController()
+    const signal = controller.signal
+    let map = await iamap.create(store, { hashAlg: 0x23 /* 'murmur3-32' */ })
+    map = await map.set('foo', 'bar', { signal: controller.signal })
+    map = await iamap.load(store, map.id, undefined, { hashAlg: 0x23, signal })
+    assert.strictEqual(await map.get('foo', { signal }), 'bar')
+
+    store.getStuck()
+    const assertion = assert.isRejected(iamap.load(store, map.id, undefined, { hashAlg: 0x23, signal }), 'Aborted')
```

Comment:
You can see how I *have* to pass `hashAlg` here, even though in theory it should be fine without it.